assets/chezmoi.io/docs/user-guide/frequently-asked-questions/encryption.md

```diff
@@ -100,3 +100,54 @@ $ chezmoi add --encrypt ~/.ssh/id_rsa
 When you run `chezmoi init` on a new machine you will be prompted to enter your
 passphrase once to decrypt `key.txt.age`. Your decrypted private key will be
 stored in `~/.config/chezmoi/key.txt`.
+
+## How to migrate between encryption tools?
+
+To migrate from gpg to age encryption or vise versa, one can follow these steps:
+
+1. Make sure you have all encrypted files applied (e.g. `chezmoi apply` decrypts files and places them in their destinations).
+2. Update chezmoi configuration to use the new encryption method
+   (examples: [gpg](/user-guide/encryption/gpg), [age](/user-guide/encryption/age), [age with one-time passphrase](#how-do-i-configure-chezmoi-to-encrypt-files-but-only-request-a-passphrase-the-first-time-chezmoi-init-is-run)).
+3. Remove all encrypted files from the state via `chezmoi forget` or `chezmoi unmanage`.
+4. Add them back with `chezmoi add --encrypt`.
+
+### Example: migrate from gpg to age
+
+Update chezmoi configuration to use age encryption (with `chezmoi edit-config` or manually editing the corresponding template):
+
+```diff
+- encryption = "gpg"
+- [gpg]
+-     recipient = "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
++ encryption = "age"
++ [age]
++     recipient = "age1xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
++     identity = "/home/user/key.txt"
+```
+
+Re-encrypt files:
+
+```shell
+#!/bin/bash
+
+for encrypted_file in $(chezmoi managed --include encrypted --path-style absolute); do
+  # optionally, add --force to avoid prompts
+  chezmoi forget "$encrypted_file"
+
+  # strip the .asc extension
+  decrypted_file="${encrypted_file%.asc}"
+
+  chezmoi add --encrypt "$decrypted_file"
+done
+```
+
+Depending on your chezmoi configuration, the above operation may lead to a number of commits in the state repository.
```

Comment:
I'm not sure that this part is required, as it entirely depends on the setting of `git.autoCommit`.

Reply:
Yes, I also wasn't entirely sure about that part. Let's proceed without it.